clients/venice-pulsar/build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id 'io.github.lhotari.gradle-nar-plugin'
+}
+
+dependencies {
+    // Schema related dependencies
+    implementation (project(':internal:venice-common')) {
+        exclude module: 'kafka_2.10'
+        exclude group: 'org.scala-lang'
+    }
+    // https://projectlombok.org
+    compileOnly 'org.projectlombok:lombok:1.18.26'
```

Comment:
How attached are you to lombok? Am not thrilled about depending on it, as I heard a fair bit of drama surrounding it (various runtime / classpath issues). Maybe it's fine for a standalone thing but it carries a reputation of being kind of toxic for libraries. Not sure how warranted that is, TBH, but it is what I observed. What's your take?

Reply:
Lombok is heavily used in Bookkeeper and Pulsar so I am used to it and it does help remove a lot of boilerplate code.
Being new to this project I'll roll with your rules and can remove lombok if that's the project's choice.

Reply:
Thanks! I have no doubt that Lombok is a powerful tool when wielded properly, but I am not sure that it is desirable to put the burden of wielding it properly on the library users (for service code, maybe that's a different decision). We can revisit this, but in the short-term, I'd feel more comfortable keeping it out.

Reply:
Library users don't have to use lombok assuming they don't build from the source.
Lombok kinda generates the code and is not needed once you get compiled jar. 
e.g. https://projectlombok.org/setup/gradle 
> To set up lombok with any build tool, you have to specify that the lombok dependency is required to compile your source code, but does not need to be present when running/testing/jarring/otherwise deploying your code. 

Developers writing the library will have to configure their IDE (once). You can see how much boilerplate code lombok saves you from writing in the diff.